### ISSUE TITLE
Reverse Proxy Doesn't Support Array Parameters

### ISSUE THREAD

**Issue by neraath** [Azure/service-fabric-issues#108]
In ASP.Net Web API, I can specify that my `GET` can accept an array of parameters from the query string by defining my controller as follows:

```
public async Task<IHttpActionResult> Get([FromUri] List<Guid> fetchIds)
```

My raw query string looks as follows:

```
/api/controller?fetchIds=GUID&fetchIds=GUID&fetchIds=GUID
```

When connecting directly to an ASP.Net Web API hosted in the cluster, this works as expected. `fetchIds` is a collection of 3 GUIDs. However, when routing through the reverse proxy, only *one* Guid makes it into the collection. All others are truncated and removed. 

When I look at the `Request.PathAndQuery` method, it looks like the following:

```
/api/controller?fetchIds=55ec98f3-3055-d9ca-8814-7eca9359865d
```

In short, Reverse Proxy is truncating my query string.

**Comment by BharatNarasimman** [comment]
Thanks for reporting this. This is a bug in our reverse proxy. I have filed an item to track this issue in our bug database.

**Comment by senj** [comment]
Is this part of the 5.4 release?

**Comment by kavyako** [comment]
This is fixed in 5.5 release.
